ionic_user_interface/src/components/wall-image-viewer/Canvas.vue

```diff
@@ -155,6 +155,18 @@ export default defineComponent({
     const isLoading = ref(false);
     const routeMakingStep = ref(0);
 
+    const computeDimensions = (heightToWidthRatio: number) => {
+      const windowHeight =
+        window.innerHeight && document.documentElement.clientHeight
+          ? Math.min(window.innerHeight, document.documentElement.clientHeight)
+          : window.innerHeight ||
+            document.documentElement.clientHeight ||
+            document.getElementsByTagName('body')[0].clientHeight;
+      const height = Math.min(props.width * heightToWidthRatio, windowHeight * 0.95);
```

Comment:
`0.85` seems good. It ll make sure not much is covered by navbar or gymselector buttons

Reply:
It's pretty small on my computer

Reply:
haiz file LOL